### PR TITLE
Added description of XMLAPI: corp/Standings.xml.aspx

### PR DESCRIPTION
**Pull request by hojerst**

### CODE REVIEW

File: docs/xmlapi/corporation/corp_standings.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Standings
-Returns the standings of non player entities have to the corporation.
+Returns the standings that non-player entities have to the corporation.
```
Comment (polarina) [inline docs/xmlapi/corporation/corp_standings.md]: To me, this reads as returning standings NPC characters have to your corporation. Can you consider rephrasing?

Comment (hojerst) [inline docs/xmlapi/corporation/corp_standings.md]: Isn't this exactly the case? The api returns the standings that factions/corps/agents have to the player corporation of the key. Or have i misunderstood the api?

Comment (polarina) [inline docs/xmlapi/corporation/corp_standings.md]: No, I am a complete derp. It looks good. Sorry. :)

**Comment by hojerst** [comment]
As a side note: Do you prefer a pull request per single api call/formula/isolated subject, or do you prefer a "pack of api calls"? (for example a 5 "corporation" api changes)

**Comment by polarina** [comment]
You can do a single pull request with many endpoints, but it would be highly preferred if you could make each endpoint a separate commit.

**Comment by JimiC** [comment]
As polarina said it's either a PR per endpoint or one PR with each endpoint as a separate commit. And the reason is that it makes code reviewing easier.

**Comment by JimiC** [comment]
Does this PR need any more work? Can we merge it?

**Comment by Regner** [comment]
Looks good. Sorry about that, it got lost in my notes to review it. :)

**Pull request merged**